ajax/libs/griddle-react/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
@pvnr0082t this could be your practice to write a test to make sure the order in `package.json`, we don't need to reach a perfect status by one step, maybe just make sure the most important things working first.